common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -477,4 +477,17 @@ public void stopYubiKitManagerUsbDiscovery() {
         }
     }
 
+    /**
+     * A wrapper to telemet results from certificate based authentication (CBA) if CBA occurred.
+     * @param response a RawAuthorizationResult object received upon a challenge response received.
+     */
+    public void telemetCertBasedAuthResult(@NonNull final RawAuthorizationResult response) {
+        final String methodTag = TAG + ":telemetCertBasedAuthResult";
+        if (mClientCertAuthChallengeHandler != null) {
+            mClientCertAuthChallengeHandler.telemetCertBasedAuthResults(response);
+        } else {
+            Logger.error(methodTag, "CBA results won't be telemeterized due to mClientCertAuthChallengeHandler being null", null);
```

Comment:
I'm not sure I would log this as an error.  When would this be null in practice?

Reply:
I _think_ it could be null if some YubiKey init method within the `ClientCertAuthChallengeHandler` constructor failed. But I haven't seen this happen yet. iirc I was asked to add logging for these types of situations (for `onReceivedClientCertRequest` and `stopYubiKeyManagerUsbDiscovery`) in a previous review.